src/main/kotlin/no/digdir/catalog_comments_service/security/SecurityConfig.kt

```diff
@@ -0,0 +1,46 @@
+package no.digdir.catalog_comments_service.security
+
+import org.springframework.boot.autoconfigure.security.oauth2.resource.OAuth2ResourceServerProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator
+import org.springframework.security.oauth2.jwt.*
+
+@Configuration
+open class SecurityConfig : WebSecurityConfigurerAdapter() {
+    override fun configure(http: HttpSecurity) {
+        http.csrf().disable()
+
+        http.cors()
+            .and()
+                .authorizeRequests()
+                .antMatchers(HttpMethod.OPTIONS)
+                    .permitAll()
+                .antMatchers(HttpMethod.GET,"/ping")
+                    .permitAll()
+                .antMatchers(HttpMethod.GET,"/ready")
+                    .permitAll()
+                .anyRequest()
+                    .authenticated()
+            .and()
+                .oauth2ResourceServer()
+                    .jwt()
+    }
+
+    @Bean
+    open fun jwtDecoder(properties: OAuth2ResourceServerProperties): JwtDecoder? {
+        val jwtDecoder = NimbusJwtDecoder.withJwkSetUri(properties.jwt.jwkSetUri).build()
+        jwtDecoder.setJwtValidator(
+            DelegatingOAuth2TokenValidator(
+                listOf(
+                    JwtTimestampValidator(),
+                    JwtIssuerValidator(properties.jwt.issuerUri),
+                    JwtClaimValidator("aud") { aud: List<String> -> aud.contains("concept-catalogue") }
```

Comment:
```suggestion
                    JwtClaimValidator("aud") { aud: List<String> -> aud.contains("catalog-comment-service") }
```
Endrer kanskje det etterhvert?